session_rpcserver.go

```diff
@@ -84,6 +85,13 @@ func (s *sessionRpcServer) AddSession(_ context.Context,
 // resumeSession tries to start an existing session if it is not expired, not
 // revoked and a LiT session.
 func (s *sessionRpcServer) resumeSession(sess *session.Session) error {
+	localPubKey := sess.LocalPublicKey.SerializeCompressed()
+
+	pubKey, err := btcec.ParsePubKey(localPubKey, btcec.S256())
```

Comment:
Why do we parse the public key here again? Shouldn't this instead just be a nil check?
Or even just:
```
pubKey := sess.LocalPublicKey
pubKeyBytes := pubKey.SerializeCompressed()
```

Reply:
ya i have no idea why i did that 🙈